### ISSUE TITLE
running .iso's by command line

### ISSUE THREAD

**Issue by lattice0** [cryptax/androidre#7]
Hello, a year ago I compiled the android emulator myself, and tried to run images by passing the .iso's myself, and I got a bug, something about C++ and the args being passed to the VM. I reported to Google but it was very low priority for them to fix it. They seemed to only support running the image you just built by compiling Android.

Did you find a way to run arbitrary images on the android emulator by command line?

**Comment by cryptax** [comment]
I'm afraid I never even tried that! What's the supposed command? using `mount` ?

**Comment by lattice0** [comment]
Here's the bug report:

https://issuetracker.google.com/issues/180021379

I develop inside VMs, but it's a pain to run the Android Emulator inside the VM so I tried to run it outside, and connect from the VM via tcp. So I tried to run the emulator without Android Studio stuff and etc. For that I compiled Google's fork of Qemu that essentially is the Android Emulator and tried to specify the .img files by command line but C++ crashed. 

I'd love to know if you got it working with .img files by command line, because I need it so much. I didn't have time back then to fix the issue but I identified where it was segfaulting if I remember correctly.
